crates/cargo-test-support/src/lib.rs

```diff
@@ -1214,6 +1228,7 @@ pub trait TestEnv: Sized {
             // stable channel yet. Once incremental support hits the stable compiler we
             // can switch this to one and then fix the tests.
             .env("CARGO_INCREMENTAL", "0")
+            .env("__CARGO_TEST_CRATESIO_REPLACEMENT", "true")
```

Comment:
See the above function `allow_silent_crates_io_replacement`.

It opts-in to the old behavior of replacing of crates.io with source-replacement including sending the crates.io token to the replacement. It was necessary for tests that do anything with `registry.token` since we don't want to send it to the real crates.io.

Reply:
The other places are using `__CARGO_TEST_ALLOW_CRATES_IO_REPLACEMENT_DO_NOT_USE` this one is using `__CARGO_TEST_CRATESIO_REPLACEMENT`

Reply:
This was a mistake. It doesn't do anything.